bundles/org.openhab.binding.mielecloud/src/main/java/org/openhab/binding/mielecloud/internal/webservice/api/json/Type.java

```diff
@@ -42,7 +42,8 @@ public Optional<String> getKeyLocalized() {
     }
 
     public DeviceType getValueRaw() {
-        return Optional.ofNullable(valueRaw).orElse(DeviceType.UNKNOWN);
+        DeviceType valueRaw = this.valueRaw;
+        return valueRaw != null ? valueRaw : DeviceType.UNKNOWN;
```

Comment:
This can be a one-liner using `Objects.requireNonNullElse(..)`

Reply:
doh... fixed.